### PR TITLE
Change logic, UI and tests of the types of jobs of a professional after login PR merge

### LINKED ISSUE #120: Change logic and tests of the types of jobs of a professional after login PR merge

### PR DESCRIPTION
**Pull request by idosinger98**
I adjusted the feature design to be part of the system design.

I added in the tests the option to check whether a user is connected or not.

I removed the typeOfJob_confirm_delete.html and when the user clicks on the delete button it does not move him to another page. Instead, it simply removes the object and stays on the same page.
Resolve #120 
image
image

### CODE REVIEW

**Review by guybeckenstein — APPROVED** [review]
LGTM

**Review by TalReinfeld — APPROVED** [review]
LGTM

File: reservation/test_typeOfJob_addList.py [hunk 1]
```diff
@@ -15,12 +15,14 @@ def save_type_of_job(make_typeOfJob):
 @pytest.mark.django_db
 class TestTypeOfJob:
     def test_typeOfJob_list(self, client, professional):
-        response = client.get(reverse('typeOfJob', kwargs={'professional': professional.pk}))
+        client.force_login(professional.profile_id.user_id)
+        response = client.get(reverse('typeOfJob'))
         assert response.status_code == 200
         assert 'reservation/typeOfJob_list.html' in response.templates[0].name
```
Comment (Yarboa) [inline reservation/test_typeOfJob_addList.py:21]: Do'nt you think you could add a test for checking there is content?
https://github.com/redhat-beyond/Pro4U/pull/129/files#diff-e5871311eb0c98a9a7e45d4efd8addcc267d4ec3ada9c3eed75d0cf73940b8bfR24

**Review by Yarboa — COMMENTED** [review]
LGTM
Please see comment

File: reservation/test_typeOfJob_addList.py (hunk 1 above)
Comment (idosinger98) [reply reservation/test_typeOfJob_addList.py:22]: Done @Yarboa

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**